### DISCUSSION TITLE
Feature Request: iOS App Notification Sound

### DISCUSSION THREAD

**Discussion by hnt3b453** [balzack/databag#20] [Ideas]
Okay! Finally getting back to this after a busy few weeks. I've been using Databag with some folks and so far it's been great!

One of the things I've noticed missing from iOS is a notification sound. I get visual notifications (lock screen, notification center, banners), but no sound, even though sound is enabled.

Not sure if this is a bug or a feature that's not implemented yet, but I'd love a notification sound.

A secondary request would be to be able to specify notification sounds for different chats!

Love the work you're doing!!!

**Comment by balzack** [comment]
Awesome feedback...thank you!

I see the issue with no sound for iOS, i committed a fix. It's a change to the server side, so I will deploy a new container tonight.

It seems like there's a way to customize the sound for notifications, but it's more involved. I'll add that work to the backlog.

**Reply by hnt3b453** [reply]
Fabulous! Just pulled the new container and tested, and notification sounds are there. Thank you!

**Comment by prologic** [comment]
Just came here to say that I installed the server and mobile app recently, but I'm not seeing notifications for iOS

**Comment by balzack** [comment]
The sound notifications can be disabled in three locations:
1] the profile tab "enable notifications"
2] admin dashboard "enable push notifications"
3] ios app notifications "allow notifications"

Can you check that all 3 of those are enabled? Once those are set, you should be able to receive a notification if a different contact starts a topic with you, adds a message to a shared topic, or starts a webRTC call with you.

Some people have reported issues running tor vpn with unified push. Could this be the case for you?

**Comment by kode54** [comment]
I was seeing no notification sounds on iOS, with all the default settings, except for granting notification permissions when prompted. Though I may have neglected to add it to my DnD bypass, and it was a night time period.

**Comment by balzack** [comment]
I will take a look and add it to the release after this upcoming one. I am already preparing a mobile release with a rebuilt UI later this month. Thanks for reporting!
